occupancy-quickstart/src/actions/userDefinedFunctions/multiplemotionsensors.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Sample code for detecting motion in an area with multiple motion sensors.
+
+var motionType = "Motion";
+var spaceAvailFresh = "AvailableAndFresh";
+
+function process(telemetry, executionContext) {
+    try {
+       // Log SensorId and Message
+       log(`Sensor ID: ${telemetry.SensorId}. `);
+       log(`Sensor value: ${JSON.stringify(telemetry.Message)}.`);
+
+       // Get sensor metadata
+       var sensor = getSensorMetadata(telemetry.SensorId);
+       
+       // Retrieve the sensor reading
+       var parseReading = JSON.parse(telemetry.Message);
+       
+       // Set the sensor reading as the current value for the sensor.
+       setSensorValue(telemetry.SensorId, sensor.DataType, parseReading.SensorValue);
+       
+       // Get parent space
+       var parentSpace = sensor.Space();
+       
+       // Get all children sensors of the parent space
+       var allSensors = parentSpace.ChildSensors();
+
+       // Get all motion sensors of the parent space
+       var motionSensors = allSensors.filter(function(item){
+         return item.DataType === motionType;
+       });
+
+       // Is any of the motion sensors detecting motion?
+       var motionDetected = motionSensors.find(function(element) {
+         return element.Value().Value.toLowerCase() === "true";
```

Comment:
this UDF makes the assumption that the sensor value that will be set on the sensors is either 'true' or 'false' - maybe worth calling that out somewhere in the comments that it is assumed parsedReading.SensorValue is either 'true' or 'false'.

Reply:
The same assumption exists in the other UDF samples of this repo. Since this is supposed to work with the existing sample code, we need not elaborate here.